src/components/TextField/tests/TextField.test.tsx

```diff
@@ -164,6 +165,21 @@ describe('<TextField />', () => {
       expect(type).toBe('email');
     });
 
+    it('sets the enableTextCouter on the input', () => {
+      const enableTextCouter = mount(<TextField label="TextField" enableTextCouter={true} />).find('input').prop('enableTextCouter');
+      expect(typeof enableTextCouter).toBe('boolean');
+    });
+
+    it('sets the counterText on the input', () => {
+      const counterText = mount(<TextField label="TextField" counterText={'Text'} />).find('input').prop('counterText');
+      expect(typeof counterText).toBe('string');
+    });
+
+    it('sets the maxLength on the input', () => {
+      const maxLength = mount(<TextField label="TextField" maxLength={100} />).find('input').prop('maxLength');
+      expect(typeof maxLength).toBe('number');
+    });
+
```

Comment:
Let's write two more tests that checks element by ID={${id}Counter} text:
1. When no max length.
2. When max length specified.